src/error/EntityNotFoundError.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Thrown when no result could be found in methods which are not allowed to return undefined or an empty set.
+ */
+export class EntityNotFoundError extends Error {
+    name = "EntityNotFound";
+
+    constructor() {
+        super();
+        Object.setPrototypeOf(this, EntityNotFoundError.prototype);
+        this.message = `No such entity.`;
```

Comment:
We need to provide a better message. Something like `No entity "${entityClass.constructor.name}" was found by following criteria: "${idOrOptionsOrConditions}` ?

Reply:
@pleerock please help me. i print entityClass from exception (EntityNotFoundError) but it is undefined.
` return this.userRepository.findOneOrFail(id);`